packages/sit-onyx/src/components/OnyxTable/OnyxTable.vue

```diff
@@ -159,6 +198,24 @@ $border: var(--onyx-1px-in-rem) solid var(--onyx-color-base-neutral-300);
     contain: paint;
     width: 100%;
 
+    &__empty {
+      display: none;
+
+      &-content {
+        display: flex;
+        justify-content: center;
+      }
+      .onyx-icon {
+        color: var(--onyx-color-text-icons-neutral-medium);
+      }
+    }
+    &:not(:has(tbody tr)) {
```

Comment:
This does not work if the project only provides <tr> without <tbody>

Maybe we should check for `<td>` instead?

![image](https://github.com/SchwarzIT/onyx/assets/67898185/634b9770-5103-4739-ae09-55515f5a4780)